src/UI/Implementation/Component/Button/Close.php

```diff
@@ -5,8 +5,10 @@
 namespace ILIAS\UI\Implementation\Component\Button;
 
 use ILIAS\UI\Component as C;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
 use ILIAS\UI\Implementation\Component\JavaScriptBindable;
 
 class Close implements C\Button\Close {
 	use JavaScriptBindable;
+	use ComponentHelper;
```

Comment:
since all components now really need the helper, should we move from trait to base class?